docs/docs/getting-started.server.mdx

```diff
@@ -481,6 +481,42 @@ Which in turn lets us choose whether to use the `@mdx-js/mdx` or
 
 ### Site generators
 
+#### Astro
+
+[Astro](https://astro.build/) has an [official MDX integration](https://docs.astro.build/guides/integrations-guide/mdx/).
+Astro recommends using [the `astro add` CLI](https://docs.astro.build/en/guides/integrations-guide/#automatic-integration-setup)
+to install dependencies and update your config automatically:
+
+```sh
+# Using npm
+npx astro add mdx
+# Using yarn
+yarn astro add mdx
+# Using pnpm
+pnpx astro add mdx
+```
+
+This base setup allows you to import Markdown,
+[Astro components](https://docs.astro.build/en/core-concepts/astro-components/),
+and other MDX files as components.
+
+**To add support for [other UI framework components](https://docs.astro.build/en/core-concepts/framework-components/)
+(e.g. Preact, Vue, Svelte and more) in your MDX,**
+you can install each as a separate integration:
+
+```sh
+# Using npm
+npx astro add preact
+# Using yarn
+yarn astro add preact
+# Using pnpm
+pnpx astro add preact
+```
```

Comment:
```suggestion
Optionally, if you haven’t already, you can also add Astro integrations for
frameworks such as React, Preact, Vue, Svelte, and more.
Each integration lets you use components written in that framework from within
MDX files.
See Astro’s
[Framework components](https://docs.astro.build/en/core-concepts/framework-components/)
guide for more information.
```

(Somewhat related due to @sarah11918’s earlier comment, which I agree with!)

I’m not sure about focussing on the Preact example (L505-L514) here. It feels “unfair”, especially due to Astro supporting so many frameworks (and not needing any), to highlight one specific framework (without explaining why).
We could show all, but that quickly becomes too much.
Or we show one, and explain it can be swapped out, but still due to several package managers being shown, we now have a “matrix of problems” 😅.
So, I’m side-stepping that problem here only pointing to the docs.

In this suggestion I also stress that it is optional to use these. As that is something Astro stresses as well, but also because some MDX users incorrectly believe MDX only works with React.

Note: in my above example, I am not sure if the second sentence is true: could you explain whether it is? And, whether MDX files in Astro, regardless of framework integration, are seen as Astro components, due to Astro’s own JSX runtime being passed to `mdx.compile`?

Reply:
Understood! Totally okay with this change 👍